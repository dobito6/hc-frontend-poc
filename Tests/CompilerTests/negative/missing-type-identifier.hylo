//! no-std

trait A { type }
