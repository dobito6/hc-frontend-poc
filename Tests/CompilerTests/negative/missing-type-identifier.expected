negative/missing-type-identifier.hylo:1.15: error: expected identifier
trait A { type }
              ^
